src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  align-content: space-around;
   margin: 0;
 }
+
+.logo.png {
```

Comment:
Avoid using tag names for styling. Use class selectors instead. The `.logo.png` looks like a filename, not a class name.

Reply:
changed  name class .logo